sss/geoid.py

```diff
@@ -65,41 +68,55 @@ def geo_identifier_creator(county_table, cpi_table):
 
     xl = pd.ExcelFile(county_table)
     # n_sheets = len(xl.sheet_names)
-    df_l = pd.read_excel(county_table, sheet_name = 0, dtype=str)
-    print("Read the first sheet of COUNTY table"+' '+ xl.sheet_names[0]+'. \
-        Please adjust the sheet order if your tagret table is not the first one')
+    df_l = pd.read_excel(county_table, sheet_name=0, dtype=str)
+    print(
+        "Read the first sheet of COUNTY table" + " " + xl.sheet_names[0]+'.\
+            Please adjust the sheet order if your tagret table is not first')
     df_l['state_fips'] = df_l['fips2010'].str[:2]
     df_l['county_fips'] = df_l['fips2010'].str[2:5]
     df_l['place_fips'] = df_l['fips2010'].str[5:]
-    
 
     xl_r = pd.ExcelFile(cpi_table)
     # n_sheets_r = len(xl_r.sheet_names)
-    df_r = pd.read_excel(cpi_table, sheet_name = 0)
-    print("Read the first sheet of CPI table"+' '+ xl_r.sheet_names[0]+'.\
-         Please adjust the sheet order if your tagret table is not the first one')
+    df_r = pd.read_excel(cpi_table, sheet_name=0)
+    print("Read the first sheet of CPI table"+' ' + xl_r.sheet_names[0]+'.\
+         Please adjust the sheet order if your tagret table is not first')
 
     try:
-        df_combine = df_l.merge(df_r,left_on = 'state_alpha', right_on = 'USPS Abbreviation', how='left')
-        df_combine = df_combine.rename(columns={"state_alpha": "state", "CPI Region": "cpi_region","SSS_Place":"place"})
+        df_combine = df_l.merge(
+            df_r,
+            left_on='state_alpha',
+            right_on='USPS Abbreviation',
+            how='left')
+        df_combine = df_combine.rename(
+            columns={
+                "state_alpha": "state",
+                "CPI Region": "cpi_region",
+                "SSS_Place": "place"})
         if df_combine.shape[0] != df_l.shape[0]:
-            print('Merge sucessfully but new rows were created due to duplications in right(CPI) table')
+            print(
+                'Merge sucessful, but new rows were created due to \
+                    duplications in right(CPI) table')
     except:
-        raise ValueError('Cannot merge, please check the two columns are named as "state_alpha" and "USPS Abbreviation"')
-
-    if df_combine.duplicated(['state','place']).sum()>0:
-        places = df_combine.loc[df_combine.duplicated(['state','place'],keep=False),'place'].values
-        counties = df_combine.loc[df_combine.duplicated(['state','place'],keep=False),'countyname'].values
-        value_com = places +' (' + counties +')'
-        df_combine.loc[df_combine.duplicated(['state','place'],keep=False),'place'] = value_com
+        raise ValueError(
+            'Cannot merge, please check the two columns are named as \
+                "state_alpha" and "USPS Abbreviation"')
+    print(df_combine.head())
```

Comment:
I think this was probably for debugging, I don't think we want to print the head regularly.